src/core/opamStubs.mli

```diff
@@ -165,3 +165,12 @@ val getVersionInfo : string -> win32_version_info option
 
 val get_initial_environment : unit -> string list
 (** Windows only. Returns the environment which new processes would receive. *)
+
+type uname = {
+  sysname : string;
+  release : string;
+  machine : string;
+}
+
```

Comment:
Does these fields need to be documented ?

Reply:
The command helps, but it is better to have also some human understandable description :)